docs/api.md

```diff
@@ -0,0 +1,191 @@
+# Go API to Run Flogo Application
+
```

Comment:
I would add  some a section describing the APIs themselves before diving into examples.